locale/en.po

```diff
@@ -3486,6 +3384,302 @@ msgstr ""
+msgid "MORE_INFO"
+msgstr "Show More Info"
+
+#: ../src/modding/ModManager.tscn:484
+msgid "OPEN_MOD_URL"
+msgstr "Open Info URL"
+
+#: ../src/modding/ModManager.tscn:508
+msgid "BROWSE_WORKSHOP"
+msgstr "Browse Workshop"
+
+#: ../src/modding/ModManager.tscn:515 ../src/modding/ModUploader.tscn:35
+msgid "UPLOAD"
+msgstr "Upload"
+
+#: ../src/modding/ModManager.tscn:522
+msgid "NEW"
+msgstr "New"
+
+#: ../src/modding/ModManager.tscn:535
+msgid "MOD_LOAD_UNLOAD_CAVEATS"
+msgstr ""
+"Note: many mods require game restart to load or unload properly. Only load "
+"mods you trust as they may contain executable code."
+
+#: ../src/modding/ModManager.tscn:561
+msgid "ENABLE_ALL_COMPATIBLE"
+msgstr "Enable All Compatible Mods"
+
+#: ../src/modding/ModManager.tscn:570
+msgid "APPLY_CHANGES"
+msgstr "Apply Changes"
+
+#: ../src/modding/ModManager.tscn:581
+msgid "UNAPPLIED_MOD_CHANGES_DESCRIPTION"
+msgstr "You need to apply your changes to load or unload any mods."
+
+#: ../src/modding/ModManager.tscn:582
+msgid "DISCARD_CHANGES"
+msgstr "Discard Changes"
+
+#: ../src/modding/ModManager.tscn:589
+msgid "FULL_MOD_INFO"
+msgstr "Full Mod Info"
+
+#: ../src/modding/ModManager.tscn:630 ../src/modding/NewModGUI.tscn:119
+msgid "MOD_NAME"
+msgstr "Mod Name:"
+
+#: ../src/modding/ModManager.tscn:718 ../src/modding/NewModGUI.tscn:211
+msgid "MOD_EXTENDED_DESCRIPTION"
+msgstr "Mod Long Description:"
+
+#: ../src/modding/ModManager.tscn:740 ../src/modding/NewModGUI.tscn:235
+msgid "MOD_ICON_FILE"
+msgstr "Icon File:"
+
+#: ../src/modding/ModManager.tscn:762 ../src/modding/NewModGUI.tscn:258
+msgid "MOD_INFO_URL"
+msgstr "Mod Info URL:"
+
+#: ../src/modding/ModManager.tscn:784 ../src/modding/NewModGUI.tscn:281
+msgid "MOD_LICENSE"
+msgstr "Mod License:"
+
+#: ../src/modding/ModManager.tscn:806 ../src/modding/NewModGUI.tscn:304
+msgid "MOD_RECOMMENDED_THRIVE"
+msgstr "Recommended Thrive Version:"
+
+#: ../src/modding/ModManager.tscn:828 ../src/modding/NewModGUI.tscn:327
+msgid "MOD_MINIMUM_THRIVE"
+msgstr "Minimum Required Thrive Version:"
+
+#: ../src/modding/ModManager.tscn:850 ../src/modding/NewModGUI.tscn:350
+msgid "MOD_MAXIMUM_THRIVE"
+msgstr "Maximum Supported Thrive Version:"
+
+#: ../src/modding/ModManager.tscn:872 ../src/modding/NewModGUI.tscn:373
+msgid "MOD_PCK_NAME"
+msgstr "Mod .pck File:"
+
+#: ../src/modding/ModManager.tscn:894 ../src/modding/NewModGUI.tscn:396
+msgid "MOD_ASSEMBLY"
+msgstr "Mod Assembly:"
+
+#: ../src/modding/ModManager.tscn:916 ../src/modding/NewModGUI.tscn:419
+msgid "MOD_ASSEMBLY_CLASS"
+msgstr "Mod Assembly Main Class:"
+
+#: ../src/modding/ModUploader.cs:235
+msgid "SAVING_DATA_FAILED_DUE_TO"
+msgstr "Saving information failed due to exception: {0}"
+
+#: ../src/modding/ModUploader.cs:268
+msgid "WORKSHOP_TERMS_OF_SERVICE_NOTICE"
+msgstr ""
+"By submitting this item, you agree to the workshop [color=#3796e1]"
+"[url=https://steamcommunity.com/sharedfiles/workshoplegalagreement]terms "
+"of service[/url][/color]"
```

Comment:
yeah you need to do some code to make it work: https://docs.godotengine.org/en/stable/tutorials/gui/bbcode_in_richtextlabel.html#doc-bbcode-in-richtextlabel-handling-url-tag-clicks

Reply:
I tested the default rich text and that worked fine. So for some reason our custom rich text class doesn't work with links automatically.

Reply:
@Kasterisk could you help me out and look into why our custom rich text doesn't work with links in it? Unless I was hallucinating from overworking the default rich text opened the url when I clicked on it, but apparently then our custom one doesn't.

Reply:
I was quite similarly dumbfounded, printing the parsed bbcode result looks fine and there's nothing wrong with it... I'll start looking into this but in the meantime maybe we should use the default rich text label exclusively in here in case it takes a while to fix.

Reply:
I would have used that, but the font for that is entirely black and different size so, I didn't use it...
Could you fix up the theme for the default rich text to make it look same as the custom rich text?

Reply:
Oh and another thing I forgot is that the custom rich text label actually automatically computes a proper height, with the inbuilt one I need to guess and add horizontal expand, otherwise it won't render at all.

Reply:
Okay so after some test the default RTL doesn't seem to also work automatically with the links for me? If it works for you then maybe in Linux it handles things differently? I guess the only fix is to go with handling the link click manually as in the provided docs https://docs.godotengine.org/en/stable/tutorials/gui/bbcode_in_richtextlabel.html#doc-bbcode-in-richtextlabel-handling-url-tag-clicks

Reply:
Looking at the docs, registering *one* handler that detects thing starts with "http" and passes it to shell open, should be pretty fast to do.

Reply:
Implemented that, and yes, it was very fast to do: https://github.com/Revolutionary-Games/Thrive/pull/2786/commits/1b1ee0693f4d3fc3e3d61413b3678b7932a18994